anx/provider/loadbalancer/address/prefix.go

```diff
@@ -51,7 +56,11 @@ func (p prefix) allocateAddress(ctx context.Context, fam v1.IPFamily) (net.IP, e
 
 	// XXX: replace this with IPAM address allocation logic once we can add and remove LoadBalancer IPs
 	// See SYSENG-918 for more info.
-	ip := calculateVIP(p.prefix)
+	ip, err := p.discoverVIP(ctx, apiClient, ipamClient, tag)
+	if err != nil {
+		log.V(1).Info("falling back to calculating canonical VIP")
```

Comment:
drop the `V(1)`, this points to incomplete migration on our side and should be more visible